<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="slicestackSubButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>480</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice / Stack -</string>
    </property>
   </widget>
   <widget class="OGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>800</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="slicestackAddButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>450</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice / Stack +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextObject">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>510</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevObject">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>510</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuAdicionar">
    <property name="title">
     <string>Adicionar</string>
    </property>
    <addaction name="actionTorus"/>
    <addaction name="actionTeapot"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuAdicionar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionTorus">
   <property name="text">
    <string>Torus</string>
   </property>
  </action>
  <action name="actionTeapot">
   <property name="text">
    <string>Teapot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">oglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
